incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/asyncapi/model/Channel.java

```diff
@@ -14,11 +14,14 @@
  */
 package io.aklivity.zilla.runtime.command.config.internal.asyncapi.model;
 
+import java.util.LinkedHashMap;
+
 import jakarta.json.bind.annotation.JsonbProperty;
 
 public class Channel
 {
     public String address;
+    public LinkedHashMap<String, Message> messages;
```

Comment:
This should just be `Map<String, Message>` right?

Reply:
It's necessary to maintain the order here for a deterministic output. 

- That's needed for the consistent run of the unit tests.
- I believe it's a good practice for the production code, too.